plugin/pkg/scheduler/factory/factory.go

```diff
@@ -370,14 +374,17 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 	}
 
 	f.Run()
-	algo := core.NewGenericScheduler(f.schedulerCache, predicateFuncs, predicateMetaProducer, priorityConfigs, priorityMetaProducer, extenders)
+	algo := core.NewGenericScheduler(f.schedulerCache, predicateFuncs, predicateMetaProducer, priorityConfigs, priorityMetaProducer, extender)
 	podBackoff := util.CreateDefaultPodBackoff()
+	if extBinder == nil {
```

Comment:
Honestly, I prefer not to use `nil` to check that :). But no solution, let's keep it fro this PR.